paddle/framework/channel_test.cc

```diff
@@ -82,6 +82,33 @@ TEST(Channel, ConcurrentSendNonConcurrentReceiveWithSufficientBufferSize) {
   delete ch;
 }
 
+void ReceiveEmptyChannelTest(Channel* ch) {
+  size_t current = 5;
+  std::thread t([&]() {
+    // Read from channel
+    int out;
+    ch->Receive(&out); // should block since channel is empty
+    current = 10;
```

Comment:
10 is a magical number

Reply:
Thank you for the review. 
I just realized that we already checked in the test cases : [BufferedChannelCloseUnblocksReceiversTest](https://github.com/PaddlePaddle/Paddle/blob/develop/paddle/framework/channel_test.cc#L108) and [UnbufferedChannelCloseUnblocksReceiversTest](https://github.com/PaddlePaddle/Paddle/blob/develop/paddle/framework/channel_test.cc#L204) that perform these operations already.
In that case, this PR would be redundant and maybe I can close it.